src/webgpu/api/validation/copyBetweenLinearDataAndTexture.ts

```diff
@@ -0,0 +1,95 @@
+export const description = `
```

Comment:
We just rely on the license+CONTRIBUTING from the root of the repo.

The test plan should move from the HackMD into the code, and you can leave the HackMD with just a link to the PR or the filename where the plan went.

Reply:
If this split a file in a way that's different from the test plan, either:
- tweak the test plan and split it between the multiple files
- put (only) the multiple files in a single directory together, and put the test plan in a README.txt file instead of a file description

Reply:
I created a new directory for these tests. The plan write now lists the points from the HackMD file with the new names of tests and what combinations we test against.